app/assets/stylesheets/application.css

```diff
@@ -67,3 +67,7 @@
   padding: 1rem;
```

Comment:
- [ ] It would be better if our README file is professional. To make it professional, it must include steps to run the app locally, steps to run test suite, working live demo link, correct screenshot, etc